src/modules/constants.ts

```diff
@@ -165,3 +165,6 @@ export const TOKEN_LOGOS_REPO_URL = `https://raw.githubusercontent.com/trustwall
 export const NETWORK_RELEASES_URL = `https://github.com/JoinColony/colonyNetwork/releases/tag`;
 
 export const SMALL_TOKEN_AMOUNT_FORMAT = '0.00000...';
+
+export const FEEDBACK_LINK =
```

Comment:
We usually keep links like these inside the component itself, but It's ok for it being in here as well.